qubesbuilder/plugins/source_deb/__init__.py

```diff
@@ -356,6 +356,7 @@ def run(self):
                         "debian": source_debian,
                         "packages": packages_list,
                         "source-hash": self.component.get_source_hash(),
+                        "files": [source_dsc, source_debian],
```

Comment:
similar here, there is one more tarball (orig tarball below) - add into "files" too?